src/Features/Core/Portable/CodeFixes/RoslynAssemblyAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    internal class RoslynAssemblyAttribute : Attribute { }
```

Comment:
This pretends to be a *general* tool, but it's only being used for this one purpose, and not every assembly has the attribute.